index.html

```diff
@@ -100,6 +100,17 @@ <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant distri
           </ul>
         </div>
 
+        <div class="card">
```

Comment:
Added a new card for basic Kafka tutorials.  Not sure of the name, suggestions very welcome.

@colinhicks I'll need lots of help with the formatting! Do we need a new card?

Reply:
I can help with the formatting. One option is to leave this stubbed as-is. I can do styling tweaks in a follow-up PR.

Reply:
I vote we move the Kafka 101 up to the top left corner, maybe "Apply function to data" could span a second row.

Reply:
+1 to @rspurgeon